docs/content/guides.mdx

```diff
@@ -9,3 +9,4 @@ This section explains how to accomplish common tasks in Dagster and showcases Da
 | [Versioning and Memoization](/guides/dagster/memoization)                 | This guide describes how to use Dagster's versioning and memoization features. <Experimental /> |
 | [Lakehouse](/guides/dagster/lakehouse)                                    | This guide describes how to use Dagster Lakehouse. <Experimental />                             |
 | [Lakehouse with Pandas and PySpark](/guides/dagster/multi-type-lakehouse) | This guide describes how to use Dagster Lakehouse with Pandas and PySpark. <Experimental />     |
+| [Re-execution](/guides/dagster/re-execution)                              | This guide describes how to re-execute a pipeline within Dagit.                                 |
```

Comment:
this guide also describes how to re-execute using the Python APIs.

Reply:
Updating